src/desktop/bootloader.desktop

```diff
@@ -15,6 +15,7 @@ X-SuSE-YaST-SortKey=
 X-SuSE-YaST-AutoInstResource=
 X-SuSE-YaST-AutoInstPath=install
 X-SuSE-YaST-AutoInstSchema=bootloader.rnc
+X-SuSE-YaST-Keywords=bootloader,GRUB,system
```

Comment:
maybe add also GRUB2 and EFI . Not sure if we should add also all supported archs like PPC, S390, ARM, AARCH, x86, i386 ?